<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edycja_serwisowanie</class>
 <widget class="QDialog" name="edycja_serwisowanie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTableView" name="tableView3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>511</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>181</width>
     <height>223</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Wybierz serwisanta</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo1"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_IMIEa_2">
      <property name="text">
       <string>Marka</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="u_marka">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wpisz imię&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_NAZWISKOa_2">
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="u_model">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wpisz nazwisko&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Dodaj nowe</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Edytuj...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>383</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Załaduj tabelę</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_load">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>300</y>
     <width>133</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID LOAD</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>id_load</zorder>
  <zorder>tableView4</zorder>
  <zorder>tableView3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
